src/frontends/tensorflow_common/src/op/const.cpp

```diff
@@ -16,15 +17,18 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_const_op(const NodeContext& node) {
-    auto ov_type = node.get_attribute<element::Type>("dtype");
+    auto ov_type = node.get_attribute_as_any("dtype");
     std::shared_ptr<Node> const_node;
-    if (ov_type == element::undefined) {
-        const_node = std::make_shared<UnsupportedConstant>();
+    if (!ov_type.is<ov::element::Type>()) {
+        if (ov_type.is<std::string>() && ov_type.as<std::string>() == "DT_STRING") {
+            const_node = std::make_shared<StringConstant>(node.get_attribute_as_any("value"));
+        } else {
+            const_node = std::make_shared<UnsupportedConstant>();
+        }
     } else {
         auto tensor = node.get_attribute<Tensor>("value");
         const_node = std::make_shared<Constant>(tensor.get_element_type(), tensor.get_shape(), tensor.data());
     }
-    set_node_name(node.get_name(), const_node);
```

Comment:
please do not delete this call. We need it to set tensor name.

Reply:
rebasing mistake, good catch!